.travis.yml

```diff
@@ -39,4 +42,18 @@ after_success:
       docker build -f Dockerfile -t codeforpoznan/pah-fm-frontend:latest .;
       docker push codeforpoznan/pah-fm-frontend;
       cd ..;
+
+      pip install -r backend/requirements/base.txt --target ./lambda_dependencies;
+      cd lambda_dependencies;
+      zip -r9 ../dev_pah_fm.zip .;
+      cd ../backend;
+      zip -g -r ../dev_pah_fm.zip .;
+      cd ..;
```

Comment:
So I wanted to make a suggestion here and simplify this bit, but as it turns out, zip(1) does not have the functionality that we need here (meaning, the cd trick is required).
So I'll leave that as is, and try to come up with better solution on different PR as well removing the big if statement at the top (I want to use conditionals instead).
Another thing - this shouldn't really be a single line script, because it's hard to tell when something is broken.
Generally `after_success` should contain a list of commands to execute, but we provide one very lengthy command instead, which looks a bit like an abuse of the feature.

Again, I won't block this further because I can't come up with better solution right now, but I'll come back to this later ;)